backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5734590_A_User_CC.sql

```diff
@@ -0,0 +1,91 @@
+-- 2024-09-24T19:11:58.811Z
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,583286,0,'CC_User_ID',TO_TIMESTAMP('2024-09-24 19:11:58.457000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'D','Y','CC Nutzer/Kontakt','CC Nutzer/Kontakt',TO_TIMESTAMP('2024-09-24 19:11:58.457000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
```

Comment:
please add a description indicating that this uses is always getting a mail as Cc when the user in question receives a mail